GDSC-MVJCE/gdscmvjce-website
# Feature/canvas transparency

## Pull Request: Enhance GDSC Logo Transparency

### Description

This Pull Request addresses the issue related to the GDSC logo's lack of transparency and introduces a new feature to enhance its usability.

### Changes Made

- **Fixed Issue:** Resolved the problem where the GDSC logo came with a solid white background, hindering its seamless integration into various digital materials and designs.
- **New Feature:** Added the requested feature to make the GDSC logo have a transparent canvas/background, significantly improving its adaptability and user-friendliness for branding and promotional purposes.
- **Optimized Workflow:** This change streamlines the logo integration process and eliminates the need for manual background removal, which was previously considered as an alternative solution.

### Related Issues

Fixes issue #82 

### How to Verify

To verify the changes and experience the enhanced GDSC logo:

1. Pull the latest changes from this branch.
2. Navigate to the "What is GDSC? section on landing page.
3. Observe that the logo now has a transparent canvas/background, allowing for seamless integration.

### Checklist

- [ ] The code has been thoroughly tested and verified to work as intended.
- [ ] The code adheres to our coding standards and best practices.
- [ ] The changes have been reviewed and approved by at least one team member.
- [ ] Environment variables or configurations have been securely updated if necessary.

### Reviewers

- @sid-js 
- @Vector-ops 
- @notBigO 

Please review and merge this Pull Request at your earliest convenience. Thank you for your consideration!
